ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -185,22 +185,126 @@ func (h *FaultHandler) CheckNetworkBlackHolePort() func(http.ResponseWriter, *ht
 	}
 }
 
-// TODO
 func (h *FaultHandler) StartNetworkLatency() func(http.ResponseWriter, *http.Request) {
```

Comment:
nit: we should add some documentation for all the handlers for future reference.

Reply:
+1, can be done either in this PR or https://github.com/aws/amazon-ecs-agent/pull/4302 depending on which PRs we'll merge in first. Seems like there may be some conflicts between the two.

Reply:
Will add some comments for network latency handler